app/__init__.py

```diff
@@ -3,6 +3,7 @@
 from werkzeug import secure_filename
 from app.routes.auth import login_required
 from app.extensions import db
+import app.extensions as app_module
```

Comment:
@hlopezes14  Can you confirm if this is debugging code? I see that we are reimporting a file module. If so, lmk it's ok to delete this and any reference to `app_module`

Reply:
Correct, it was used to debug the promote_user function. I forgot to delete this after debugging.